### ISSUE TITLE
Error: could not download pre-built `cargo-generate`

### ISSUE THREAD

**Issue by tomByrer** [cloudflare/wrangler#1034]
#### Environment

* operating system: Win10
* output of `rustc -V`: `bash: rustc: command not found`
* output of `node -v`: `13.2.0`
* output of `wrangler -V`: `wrangler 1.7.0`
* shell: `GNU bash, version 4.4.23(1)-release (x86_64-pc-msys)` (Git Bash, maybe a few months old)

#### Steps to reproduce

```bash
> wrangler generate cfw-router https://github.com/cloudflare/worker-template-router
Error: could not download pre-built `cargo-generate` (no prebuilt cargo-generate binaries are available for this platform).
```

I got this error after having to re-install wrangler.
Then [deleted npm -global folder](https://stackoverflow.com/a/15597395/1324588), verified wrangler was gone.
Re-installed via `npm i @cloudflare/wrangler -g`
Re-ran above command, got same error.

**Comment by Cherry** [comment]
I'm seeing a very similar error when using `wrangler generate` on the latest version, also using Windows 10.

```
Error: could not download pre-built `wasm-pack` (no prebuilt wasm-pack binaries are available for this platform).
```

**Comment by EverlastingBugstopper** [comment]
Hey @tomByrer and @Cherry - we're looking into this now. Could either of you try installing with `cargo install wrangler` and see if that fixes your problem? I have a hunch this has to do with how Azure Pipelines built our latest release binary but I'm not 100% sure (I'm on MacOS, need to do some tests with CI).

**Comment by tomByrer** [comment]
**Rolled back to _v1.6.0_, seems to work fine**:

1. Removed npm folder
2. `npm install -g @cloudflare/wrangler@1.6.0`
3. worked!
 ```bash
$ wrangler -V
wrangler -V
 wrangler 1.6.0

tomby@MSI MINGW64 /d/doc/prog/tb/cfworker
$ wrangler generate cfw-router https://github.com/cloudflare/worker-template-router
wrangler generate cfw-router https://github.com/cloudflare/worker-template-router
⬇️ Installing cargo-generate...
 Creating project called `cfw-router`...
 Done! New project created D:\doc\prog\tb\cfworker\cfw-router
...etc
```

**Comment by tomByrer** [comment]
@Cherry:  Please be my guest to test out via Cargo; I have drive to a Meetup to hear Cory House talk about React stuff right now 😎

**Comment by Cherry** [comment]
@tomByrer Enjoy!

I can confirm that when installing with `cargo` and not `npm` on Windows 10, I no longer receive the error using the latest `1.7.0` version.

**Comment by EverlastingBugstopper** [comment]
Thanks @Cherry 😄 hope that unblocks you for the time being. We're working on re-building our release binary which should fix this issue, I'll update here when that work is complete.

**Comment by EverlastingBugstopper** [comment]
We've replaced the binary and installing with npm should work again. Thanks so much for the reports on this - we'll be switching our release process to use GitHub Actions in the future which _should_ prevent this from happening.

**Comment by CopernicAI** [comment]
Hi there guys. Have had clean install of cargo and  wrangler (on Windows 10) . Seeing the same error with wrangler 1.9.2
Manually installed 'cargo-generate' didn't help either.
error msg:  
******************************************************
' Installing cargo-generate v0.5.0...
Error: could not download `cargo-generate`
no prebuilt cargo-generate binaries are available for this platform'
*************************************************************
any suggestions ? thanks in advance :)

**Comment by EverlastingBugstopper** [comment]
@CopernicAI since this seems to be a new issue (although seems related to the old one) could you please open a new one so we can track there? When you do so, could you also include your operating system and version of node? Thanks!

**Comment by CopernicAI** [comment]
I tried to reinstall with npm and it is working now :) thanks! :)
